abstract.tex

```diff
@@ -0,0 +1,18 @@
+\begin{abstract}
+
+The Vera C. Rubin Observatory will revolutionize many areas of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
```

Comment:
"will advance nearly every area of astronomy"

Reply:
nice - updated